plugins/services/src/js/components/LogView.js

```diff
@@ -269,17 +269,9 @@ class LogView extends React.Component {
           {fullLog}
         </Highlight>
       </pre>,
-      <CSSTransitionGroup
-        key="log-go-down-button"
-        transitionAppear={true}
-        transitionName="button"
-        transitionAppearTimeout={350}
-        transitionEnterTimeout={350}
-        transitionLeaveTimeout={350}
-        component="div"
-      >
+      <TransitionGroup appear={true}>
```

Comment:
At some point this may have transitioned in, but there aren't currently any in/out transition styles to I could find for this one 🤷‍♂️

I'd recommend we just remove this TransitionGroup

Reply:
This is for the Go To Bottom button and I think it does transition in if I'm not mistaken.

Reply:
I checked in this branch and on master, and it just pops in - no transition. If you're seeing a transition, could you post a screen capture?

Reply:
Maybe I'm mistaken. And I can't find any styles for `button-enter` `button-appear` etc. I'll double check and if it doesn't transition I'll remove this.

Reply:
You are correct there is not transition, removing.

Reply:
The transition lives on in our imagination <3